plugin.video.presstv/default.py

```diff
@@ -0,0 +1,14 @@
+import xbmc, xbmcgui, os, xbmcaddon, xbmcplugin
+
+__settings__ = xbmcaddon.Addon()
+home = __settings__.getAddonInfo('path')
+addon_handle = int(sys.argv[1])
+icon = xbmc.translatePath(os.path.join(home, 'icon.png'))
+li = xbmcgui.ListItem ('Live Stream')
+li.setArt({ 'thumb': icon })
+li.setInfo('video', {'plot':'PressTV Live Stream'})
+
+liveVideoUrl = "https://live2.presstv.com/liveprs/smil:liveprs.smil/playlist.m3u8"
```

Comment:
Can you include parser code that scrapes this link from its origin website? THX

Reply:
@scriptik Where is this video stream hosted? In other words: where did you get the stream from?

Reply:
Link verified, okay as standalone.